google/cloud/speech/client.py

```diff
@@ -60,28 +62,19 @@ class Client(client_module.Client):
 
     _connection_class = Connection
 
-    def syncrecognize(self, content, uri, encoding, sample_rate,
-                      language_code=None, max_alternatives=None,
-                      profanity_filter=None,
-                      speech_context=None):
+    def sync_recognize(self, content, encoding, sample_rate,
+                       language_code=None, max_alternatives=None,
+                       profanity_filter=None,
+                       speech_context=None):
         """Synchronous Speech Recognition.
 
-        .. _syncrecognize: https://cloud.google.com/speech/reference/\
+        .. _sync_recognize: https://cloud.google.com/speech/reference/\
                              rest/v1beta1/speech/syncrecognize
 
-        See `syncrecognize`_.
+        See `sync_recognize`_.
 
-        :type content: str
-        :param content: The content string containing the audio
-                      data bytes encoded as specified in RecognitionConfig.
-                      This is a base64-encoded string
-
-        :type uri: str
-        :param uri: URI that points to a file that contains audio
-                      data bytes as specified in RecognitionConfig.
-                      Currently, only Google Cloud Storage URIs are
-                      supported, which must be specified in the following
-                      format: gs://bucket_name/object_name
+        :type content: bytes
+        :param content: Byte stream of audio.
```

Comment:
Why drop the `source_uri` support?  I don't think I asked for that (I did ask to have it renamed from `uri` -> `source_uri`).